src/account/index.ts

```diff
@@ -1,6 +1,7 @@
-export * from './types'
-export * from './useAccount'
-export * from './useEmailSignIn'
-export * from './useEmailSignUp'
-export * from './useOAuth2SignIn'
-export * from './useSignOut'
\ No newline at end of file
+export * from "./types"
+export * from "./useAccount"
+export * from "./useEmailSignIn"
+export * from "./useEmailSignUp"
+export * from "./useAnonSignIn"
+export * from "./useOAuth2SignIn"
+export * from "./useSignOut"
```

Comment:
Can you change these to use single quotes `'` to match the rest of the project and remove semicolons throughout? I see you made a commit to match, but it looks like you may have missed these.

Reply:
Will do! I used the default prettier configs so I disabled prettier and tried to go back and revert the changes it made. Guess I missed a few :P